app/migrations/Version20170116215417.php

```diff
@@ -20,7 +20,7 @@ class Version20170116215417 extends AbstractMigration
 {
     public function up(Schema $schema): void
     {
-        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+        $this->abortIf(!is_a($this->connection->getDatabasePlatform(), \Doctrine\DBAL\Platforms\MySqlPlatform::class, true), 'Migration can only be executed safely on \'mysql\'.');
```

Comment:
Why don't you change abstract class as in migrations from CoreBundle?

And maybe you know and you will remind me, because I don't remember, why we need these migrations here too? 🤔

Reply:
Good question as I missed the existence of this directory :D. I guess it's a legacy from the times when Symfony 3 was alive :D.

Reply:
I know that migrations existed in this directory before, but when we switched to Doctrine/Migrations 3.0, they were moved to CoreBundle and I'm just trying to remind myself the reason why they stayed here and if we still need them duplicated 🤔 I guess no, but I wanted to see the opinion of others 😃

Reply:
@lchrusciel a few migrations are missing in `app/migrations` so anybody using the old path had to switch to new path anyway 🤔 So should they still be there?

Reply:
it is not about missing - people can use them and generate their own. There was something about coping with these migrations to the end app of users IIRC. But then, we moved to the new structure, so if someone wants to use ours, needs to move on as well. 

long story short, we can remove them in Sylius 2.0 in my opinion

Reply:
So I guess it would be nice to change them, too (extend our custom `AbstractMigration` and remove if clauses), am I correct?

Reply:
Migrations in `app/migrations` are updated the same way I've updated those from `CoreBundle`.